man/systemd.exec.xml

```diff
@@ -1479,7 +1479,7 @@ CapabilityBoundingSet=~CAP_B CAP_C</programlisting>
               </row>
               <row>
                 <entry>@keyring</entry>
-                <entry>Kernel keyring access (<citerefentry project='man-pages'><refentrytitle>keyctl</refentrytitle><manvolnum>2</manvolnum></citerefentry> and related calls)</entry>
+                <entry>Kernel key ring access (<citerefentry project='man-pages'><refentrytitle>keyctl</refentrytitle><manvolnum>2</manvolnum></citerefentry> and related calls)</entry>
```

Comment:
"keyring" is a valid word nowadays.

Reply:
hmm, i checked merriem webster specifically, and it insisted on "key ring"... But i figure if it's "keyring" according to wikipedia in respect to cryptography, i figure that's fine too

Reply:
Yeah, I think it'd be confusing. Internally in the code we have `keyring` in various places, so adding `key_ring` in others is inconsistent.